src/_utils/themeProvider/index.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { ReactNode } from 'react'
+import { createGlobalStyle } from 'styled-components'
+
+import { color } from '../branding'
+
+const CDN_BASE_URL = 'https://cdn.blablacar.com/kairos/assets/fonts'
```

Comment:
```suggestion
const CDN_FONTS_BASE_URL = 'https://cdn.blablacar.com/kairos/assets/fonts'
```

Because of `assets/fonts'`, CDN could be use for other things in the future

Reply:
Done